test/model/store_test.dart

```diff
@@ -719,6 +719,30 @@ void main() {
       connection.prepare(httpStatus: 200, body: 'nonsense');
     }
 
+    void prepareRateLimitExceptionCode() {
+      // Example from the Zulip API docs:
+      //   https://zulip.com/api/rest-error-handling#rate-limit-exceeded
+      // (The actual HTTP status should be 429, but that seems undocumented.)
+      connection.prepare(httpStatus: 400, json: {
+        'result': 'error', 'code': 'RATE_LIMIT_HIT',
+        'msg': 'API usage exceeded rate limit,',
```

Comment:
nit:

```suggestion
        'msg': 'API usage exceeded rate limit',
```

From the API doc linked here.
```
{
    "code": "RATE_LIMIT_HIT",
    "msg": "API usage exceeded rate limit",
    "result": "error",
    "retry-after": 28.706807374954224
}
```

Reply:
Indeed that's a typo, thanks :slightly_smiling_face: